opentracing_instrumentation/client_hooks/psycopg2.py

```diff
@@ -31,25 +31,49 @@
 else:
     _psycopg2_connect = psycopg2.connect
     _psycopg2_extensions_register_type = psycopg2.extensions.register_type
+    _psycopg2_extensions_quote_ident = psycopg2.extensions.quote_ident
+
+
+class Psycopg2CursorClass(CursorWrapper):
+    """
+    Psycopg2 accept not only string as sql statment, but instances of
+    ``psycopg2.sql.Composable`` that should be represented as string before the
+    executing.
+    """
+    def execute(self, sql, params=NO_ARG):
+        if not isinstance(sql, str):
```

Comment:
Hm, it can be unicode too. So it's better to check is sql instance of `Composable` class.